client/me/purchases/vat-info/index.tsx

```diff
@@ -84,11 +100,58 @@ export default function VatInfoPage() {
 					</CardHeading>
 					<p className="vat-info__sidebar-paragraph">
 						{ translate(
-							/* translators: %s is the name of taxes in the country (eg: "VAT" or "GST"). */
-							"We currently only provide %(taxName)s invoices to users who are properly registered. %(taxName)s information saved on this page will be applied to all of your account's receipts.",
+							/* translators: %s is the name of taxes in the country (eg: "VAT" or "GST") or a generic fallback string of tax names */
+							'The %(taxName)s details saved on this page will be applied to all receipts in your account.',
 							{
-								textOnly: true,
-								args: { taxName: taxName ?? translate( 'VAT', { textOnly: true } ) },
+								args: { taxName: taxName ?? translate( 'tax (VAT/GST/CT)', { textOnly: true } ) },
+							}
+						) }
+						<br />
+						<br />
+						{ translate(
+							/* translators: %s is the name of taxes in the country (eg: "VAT" or "GST") or a generic fallback string of tax names */
+							/* translators: This is a list of tax-related reasons a customer might need to contact support */
+							'If you:' +
```

Comment:
I understand the intent of this UI but I do wonder if it might be hard to translate well into other languages. The construction of `If you: X / Y / X` feels like an English thing to me. I'm not sure how I'd go about translating it into Japanese [which uses verb conjugations to convey conditionals](https://storylearning.com/learn/japanese/japanese-tips/japanese-conditional-form). 🤔 

When faced with complexity like this I usually try to reconsider what I actually want to say and see if there's a simpler sentence structure that would work for the same idea.

In this case, what about something like, `If you need to change saved %(taxName) details, if you have been charged taxes as a business subject to reverse charges, or if you do not see your country listed in this form, {{contactSupportLink}}contact our Happiness Engineers{{/contactSupportLink}}.` then followed by a separate string `Include your %(taxName)s number and country code when you contact us.`?

Reply:
Thanks for the insight! We actually pulled this from our support documentation. A similar version has already been translated into Japanese here: https://wordpress.com/ja/support/vat-gst-other-taxes/#i-cant-add-my-vat-number-to-my-account

```
もし

- 既存のビジネス納税者番号情報を更新する必要がある場合
- リバースチャージの対象となる VAT 登録事業者として VAT を請求されたことがある場合
- 購入または更新を完了できない場合
- ご質問がある場合は…
```

I would imagine the string starting at line 114 would be translated similarly. Do you think the version above makes sense?

Reply:
Oh, interesting. Translated back to English, that looks something like what I suggested above, actually: making each bullet point its own full "if" sentence.

<img width="696" alt="Screenshot 2023-06-27 at 10 16 17 AM" src="https://github.com/Automattic/wp-calypso/assets/2036909/d0766945-f0fd-4934-bd92-0900f53ed22f">

I suppose the translators just figured out a way to do it. I still feel that this is a little awkward to localize but I won't consider it a blocker.